### ISSUE TITLE
Implement a solid external cover

### ISSUE THREAD

**Issue by ekspek** [ekspek/ist-thesis#5]
An extra possible feature is the implementation of an external cover as defined at the end of page 3 of [this](https://academica.tecnico.ulisboa.pt/files/sites/54/guia-de-preparacao-da-dissertacao-1516.pdf) document. 

Essentially, the original document cover needs to be complemented with a spine that must contain specific data as mentioned in the linked document. This can be implemented additionally using the class file and the macros used to make the original cover.

**Comment by jomigo96** [comment]
I don't see what is missing, apart from making the image optional.

**Comment by ekspek** [comment]
Absolutely a fair question. When thesis get print out, they're suggested to come with a back cover, besides the main cover already included. [Page 6 of the main IST guide](https://academica.tecnico.ulisboa.pt/files/sites/54/guia-de-preparacao-da-dissertacao-1516.pdf) has an example for the spine typesetting style.

The idea for implementing this in the template came from [TUDelft's report template](https://d1rkab7tlqy5f1.cloudfront.net/Websections/TU%20Delft%20Huisstijl/report_style.zip) (check the ```cover.pdf``` file in the linked zip), which implements it as a part of the ```makecover``` command like this:
```tex
\title{Title}
\subtitle{Optional subtitle}
\author{J.\ Random Author}
\affiliation{Technische Universiteit Delft}
\coverimage{cover.jpg}
\covertext{
    \textbf{Cover Text} \\
    possibly \\
    spanning \\
    multiple \\
    lines
    \vfill
    ISBN 000-00-0000-000-0
}

%% The 'back' option enables the generation of the back cover.
\makecover[back,whitelogo]
```

Bottom line is this: the idea is that it's a cover additional to the one already implemented.

**Comment by ekspek** [comment]
I just added the ```\makespine``` command to commit e8abe16df73447aba147ac6be4df2310d0e6e10c, which is a preliminary version of the idea. For now, it prints out an extra page with a cut-out of the document spine, just like the main guide.

The final idea is to put it on the ```\makecover``` command as the ```full``` option, which would output a page bigger than an A3 including the front and back covers and the spine in between. The pieces are all there, they just need to be put together.
